apps/www/registry/default/block/chart-area-interactive.tsx

```diff
@@ -149,8 +74,8 @@ export default function Component() {
       daysToSubtract = 7
     }
     now.setDate(now.getDate() - daysToSubtract)
-    return date >= now
-  })
+    return new Date(date) >= new Date(now)
```

Comment:
Why create `new Date` here?

Reply:
To get today's date. This function will get date of last 90days dynamically. 
Earlier last 90dates were added manually, so after 90 days again the date has to updated manually in code.

Reply:
`generateRandomData` istead of `chartData` makes total sense to me. 

I don't think you need to change this line though. Both `date` and `now` are already `Date` objects,  so there's no need to wrap them in `new Date(...)`.

Reply:
Removed new Date()